tests/ui/pages/desktop/collections.py

```diff
@@ -0,0 +1,31 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class Collections(Base):
+    """ Collections page. """
+
+    _root_locator = (By.CLASS_NAME, 'island')
+
+    _name_locator = (By.CSS_SELECTOR, ".item > .info > h3")
+
+    def wait_for_page_to_load(self):
+        self.wait.until(lambda _: self.find_element(
+            *self._name_locator)).is_displayed()
```

Comment:
The Problem I was having here is that usually it will error and say the index is out of range. I think its because it doesn't actually find anything and errors on the first try, as the collections page hasn't loaded yet.